### ISSUE TITLE
"has" method has a problem

### ISSUE THREAD

**Issue by wendao76** [catfan/Medoo#720]
'''
return $result === '1' || $result === true;
'''
in fact $result=1,  so the method always return false

**Comment by catfan** [comment]
What's database and version you are using that returned `$result = 1` as integer type?

**Comment by ha-ni-cc** [comment]
I have this problem,too.

Use mysql5.7 and medoo 1.5.4

**Comment by lovearthhome** [comment]
@wendao76 @catfan @ha-ni-cc  yes, it's a bug. now,just use "PDO::ATTR_EMULATE_PREPARES        => true" to solve this.
